content-api/data/requirements.txt

```diff
@@ -0,0 +1 @@
+firebase-admin==5.4.0
```

Comment:
Future note: firebase-admin is a multi-package with libraries for most of Firebase. Using the Cloud Firestore library would be more narrowly scoped and all we are likely to need.

This is out of scope of this PR, noting it only because the dependency we are currently using is being more clearly identified here.

Reply:
I updated it to use google-cloud-firestore instead